src/main/java/frc/robot/subsystems/Swerve.java

```diff
@@ -410,17 +491,15 @@ public Command getChaseCommand(Supplier<Pose2d> desiredPose) {
     }
 
     public Command getScanCommand() {
-        return new ChasePose(this, () ->
-            new Pose2d(getPose().getTranslation(), Rotation2d.fromDegrees(Robot.isRedAlliance() ? 0 : 180)).plus(
-                new Transform2d(
-                    0,
-                    0, 
-                    Rotation2d.fromRadians(
-                        Math.cos(Robot.currentTimestamp)/2.0)
-                    )
-                ), 
-                () -> false
-        );
+        return new ChasePose(this,
+                () -> new Pose2d(getPose().getTranslation(), Rotation2d.fromDegrees(Robot.isRedAlliance() ? 0 : 180))
+                        .plus(
+                                new Transform2d(
+                                        0,
+                                        0,
+                                        Rotation2d.fromRadians(
+                                                Math.cos(Robot.currentTimestamp) / 2.0))),
+                () -> false);
```

Comment:
why was our manual formatting not good enough?

Reply:
I didn't do that... I'll take a look through my settings again today.... 4th trys the charm

Reply:
i found it... finally